settings.gradle.kts

```diff
@@ -85,6 +85,9 @@ include(":examples:mcp-server")
 include(":mcp")
 include(":mcp:mcp-schemas")
 include(":mcp:mcp-server")
-
+include(":mcp:aws-mcp-provider")
```

Comment:
This should go under `:aws` instead of `:mcp`

Reply:
I'll move aws-mcp-types there too